src/css/abstracts/_constants.scss

```diff
@@ -1,15 +1,15 @@
 :root {
     --font-family-default: sans-serif;
     
-    --font-size-plus-4: 1rem;
-    --font-size-plus-3: 1rem;
-    --font-size-plus-2: 1rem;
-    --font-size-plus-1: 1rem;
+    --font-size-plus-4: 2rem;
+    --font-size-plus-3: 1.75rem;
+    --font-size-plus-2: 1.5rem;
+    --font-size-plus-1: 1.25rem;
     --font-size-baseline: 1rem;
-    --font-size-minus-1: 1rem;
-    --font-size-minus-2: 1rem;
-    --font-size-minus-3: 1rem;
-    --font-size-minus-4: 1rem;
+    --font-size-minus-1: .9rem;
+    --font-size-minus-2: .8rem;
+    --font-size-minus-3: .7rem;
+    --font-size-minus-4: .6rem;
 

```

Comment:
The font-size constants are intentionally left all as 1rem to prevent recycling typescales between projects (which has happened on rare occasions in the dark days of the past).

I realise that these sizes are for the image component content on the home page, but I wonder we should have that content? I think we should tend to leave the pages templates clear (including deleting the unhelpful grids.js page) and improving the docs. In time example code could be added to the image component docs (we have a branch and an open issue to write more docs, so it is on the horizon https://github.com/stormid/scaffold/tree/feature/82).

Reply:
@mjbp I have reset the defaults.